mapie/control_risk/ltt.py

```diff
@@ -0,0 +1,134 @@
+import warnings
+
+from typing import Tuple, List, Optional, Any
+from mapie._typing import NDArray, ArrayLike
+
+import numpy as np
+
+from .p_values import compute_hoeffdding_bentkus_p_value
+
+
+def _ltt_procedure(
+    r_hat: NDArray,
+    alpha_np: NDArray,
+    delta: Optional[float],
+    n_obs: int
+) -> Tuple[List[List[Any]], NDArray]:
+    """
+    Apply the Learn-Then-Test procedure for risk control.
```

Comment:
No need to specify where this function is called as it will change if we add new controls (and we will surely forgot to change this docstring)

Reply:
Done, does the new docstring satisfy you?